contracts/proxy-upgrade/exchange/Exchange.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.0 <0.9.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import "../../hts-precompile/IHederaTokenService.sol";
+import "../../hts-precompile/HederaResponseCodes.sol";
+
+contract Exchange is OwnableUpgradeable, UUPSUpgradeable {
+    address public tokenAddress;
+    address private precompile;
```

Comment:
Let's assign the value directly here and make it constant `address constant private precompile = address(0x167);`

Reply:
Done.